build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-turing/build.sh

```diff
@@ -45,4 +45,4 @@ echo "Building with cmake"
 ./build_tools/cmake/clean_build.sh
 
 echo "Testing with ctest"
-./build_tools/cmake/test.sh
+./build_tools/cmake/test.sh --test_env=IREE_CUDA_DISABLE=0
```

Comment:
This is a bash script for cmake. There is no `--test_env` :-P

```suggestion
export IREE_CUDA_DISABLE=0
./build_tools/cmake/test.sh
```

Reply:
Since we have to do the dance with the VM image and docker image at once and can't test on presubmit, let's avoid changing what tests run for now.